lib/ts/Layout.cc

```diff
@@ -28,6 +28,12 @@
 #include "ts/ink_string.h"
 #include "ts/I_Layout.h"
 
+#include <string>
+#include <fstream>
+#include <iostream>
+
+using namespace std;
```

Comment:
I am ok with this in the file scope of a .cc/cpp file. I am not ok with it in the scope of a .h file.
This mostly helps with writing functions in the source file. However I am not against this suggest. I am 100% for using namespaces to prevent ambiguous code in the global namespace and to help provide clarify